dbfit-java/config/checkstyle/checkstyle.xml

```diff
@@ -9,7 +9,9 @@
     <module name="UnusedImports"/>
   </module>
   <module name="FileTabCharacter"/>
-  <module name="NewlineAtEndOfFile"/>
+  <module name="NewlineAtEndOfFile">
+    <property name="lineSeparator" value="lf"/>
```

Comment:
@MMatten I think we should add *.java to .gitattributes for eol=lf. It will make things easier to follow. And yes additionally it would be good to add some basic guidance in contributing.md to make IDE configuration clear.

Reply:
We may be forcing people to use certain editors in that case. E.g. I don’t think notepad can save with LF as EOL. 

What’s the benefit of forcing new local files to be LF?

Reply:
@MMatten If I remember right I introduced this addition because I had checkstyle complaining missing EOL and EOF simply because the file was LF and not CR/LF. 

In general I tend to force everything on one Newline format for specific file types in projects because it often introduces hard to trace errors (e.g. bash scripts, powershell scripts expect a certain Newline format).

Reply:
I also prefer having stable formatting of files. It makes me more confident that the build will produce same result on different platforms.
Seems that Notepad supports saving LF style of lines when file was in that style (just creating new files might be limited). There are other editors and tools which fully support LF and CRLF and switching between them.

There is already a note in End of line style in `CONTRIBUTING.md`. It should be straightforward enough for people to figure out how to configure their own editor. I consider it out of scope of dbfit to outline exact details how this configuration is done in each possible editor.

I just pushed `*.java text eol=lf`.

In any case - I think if anyone faces issue with line endings we may easily handle it accordingly (either supporting the editor setup, removing that enforcement or some other way).

Reply:
@MMatten, @bhuesemann - are you OK merging that PR considering the latest commends and changes here (`*.java text eol=lf` added)

Reply:
I'm happy with this - looks good to me.

Reply:
me too